docs/developer/build_and_test.html

```diff
@@ -0,0 +1,188 @@
+<!--
+  EVE -  Expressive Vector Engine
+  Copyright 2021 Joel FALCOU
+  Copyright 2021 Jean-Thierry LAPRESTE
+  Copyright 2021 Denis YAROSHEVSKIY
+
+  Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+  SPDX-License-Identifier: MIT
+ -->
+
+ <meta charset="utf-8" lang="en"><style class="fallback">body{visibility:hidden;}</style>
+ **Expressive Vector Engine**
+
+(insert ../crumbs.html here)
+
+# Building EVE: how to do local development.
+
+*NOTE: this documentation was valid at [commit](https://github.com/jfalcou/eve/commit/2fe1048aa31eb3271666f1cca04bf2f5f34eeb6e) and might get outdated.* <br/>
```

Comment:
as any file, I guess this kind of disclaimer is not useful.

Reply:
I think this is a good practice in docs.